test/create_db.sh

```diff
@@ -9,6 +9,11 @@ if [[ ! -z "$MYSQL_CONTAINER" ]]; then
 	dbconn="-u root -h 127.0.0.1 --port 3306"
 fi
 
+# MariaDB sets the default binlog_format to STATEMENT,
+# which causes warnings that fail tests. Instead set it
+# to the format we use in production, MIXED.
+mysql $dbconn -e "SET GLOBAL binlog_format = 'MIXED';"
```

Comment:
Whoops, yep, it does help better reflect the prod environment but maybe should be done in another patch?